job-dsls/jobs/prod/prod_pr_jobs.groovy

```diff
@@ -0,0 +1,121 @@
+def configs = [:]
+
+configs['izpack'] = [
+        pmeParams: '''
+         -DversionOverride=${IZPACK_VERSION}
+        '''
+]
+
+configs['installer-commons'] = [
+        pmeParams: '''
+         -DversionOverride=${INSTALLER_COMMONS_VERSION}
+         -DizpackVersion=${IZPACK_VERSION}
+         -DgroovyScripts=file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerCommonsAlignment.groovy 
+        '''
+]
+
+configs['rhba-installers'] = [
+        pmeParams: '''
+         -DversionOverride=${RHPAM_VERSION}
+         -DizpackVersion=${IZPACK_VERSION}
+         -DinstallerCommonsVersion=${INSTALLER_COMMONS_VERSION}
+         -DkieVersion=${KIE_VERSION}
+         -DgroovyScripts=file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerAlignment.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerCommonsAlignment.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/ForceComRedhatBaVersion.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/ForceComRedhatBaVersionLast.groovy
+        '''
+]
+
+configs['bxms-patch-tools'] = [
+        branch: 'bxms-7.0',
+        mavenParams: '-Prhpam,rhdm,integration-tests',
+        pmeParams: '''
+         -DversionOverride=${RHPAM_VERSION} 
+         -DkieVersion=${KIE_VERSION} 
+         -DupdatableVersionRange=7.9.0.redhat-00002,7.9.1.redhat-00003,7.10.0.redhat-00004,7.10.1.redhat-00001
```

Comment:
how do we update these ranges? Isn't the maintenance a burner?

Reply:
It is. The problem is that we need them there for PME to run correctly. They are needed also to run the integration tests. Having them here outdated shouldn't be a problem as the Update Tool would just use them to create a patch and check whether it can be applied using the integration-tests. I have created the below anyway to address that ASAP, so I would merge this in now and later change that.

https://issues.redhat.com/browse/BXMSPROD-1427